packages/web-react/src/types/modal.ts

```diff
@@ -17,11 +17,12 @@ export type ModalDialogHandlingProps = {
   isOpen: boolean;
   onClose: (event: ClickEvent) => void;
   closeOnBackdropClick?: boolean;
+  closeOnEscapeKeyDown?: boolean;
 };
 
 export interface ModalCloseButtonProps extends ModalDialogHandlingProps {
   id: string;
-  label: string;
+  label?: string;
```

Comment:
is it actually optional? I don't think so

Reply:
Currently, we set the `Close` in ModalHeader and ModalCloseButton too. Can we set it only once? Then this change would be ok.

Or we can leave it as it was if it was working, but anyway that we set it twice is wrong.

Reply:
Ha, it is already like this. So LGTM!